lib/Runtime/ByteCode/ByteCodeGenerator.cpp

```diff
@@ -1426,8 +1446,12 @@ void ByteCodeGenerator::EndBindFunction(bool funcExprWithName)
     }
     else
     {
-        Assert(currentScope->GetScopeType() == ScopeType_Parameter);
-        PopScope(); // parameter scope
+        Scope* paramScope = bodyScope->GetFunc()->GetParamScope();
+        if (paramScope == nullptr || paramScope->GetCanMergeWithBodyScope())
```

Comment:
I doubt that the paramScope will be null here. In the StartBindFunction we created paramScope if it was not there. What happens in the global function?